e2e_tests/tests/config.py

```diff
@@ -196,3 +196,9 @@ def set_profiling_enabled(config: Dict[Any, Any]) -> Dict[Any, Any]:
     config.setdefault("profiling", {})
     config["profiling"]["enabled"] = True
     return config
+
+
+def set_entrypoint(config: Dict[Any, Any], entrypoint: str) -> Dict[Any, Any]:
+    config = config.copy()
+    config["entrypoint"] = entrypoint
+    return config
```

Comment:
Can you configure your editor to always write the last line of the file with a terminating newline?  That is an old unix standard, and lots of other people's editors will do that automatically, so it creates annoying git churn when some files are created not like that.